app/code/Magento/Theme/view/frontend/templates/html/sections.phtml

```diff
@@ -1,20 +1,21 @@
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Copyright 2024 Adobe
```

Comment:
Year in copyright header should be the year the file was first added in git, so in this case 2014 instead of 2024